src/components/navigation/GlobalNavigation/GlobalNavigationHelpers.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { Center, IGlobalNavigationProps, Icon } from 'src/components'
+import { Popover, Tooltip } from 'antd'
+import MiniMap from 'src/components/navigation/MiniMap/MiniMap'
+
+export function CenterContent({ onClick }: { onClick: () => void }) {
+  return (
+    <Center className="globalNavigation__mpHome" onClick={onClick}>
+      <Icon name="mpLogo" size="lg" color="white" />
+    </Center>
+  )
+}
+
+export function WithMinimap({ goToOverviewHref, onClick }: { goToOverviewHref: string; onClick: () => void }) {
+  return (
+    <Popover content={() => <MiniMap goToOverviewHref={goToOverviewHref} />} placement="rightBottom" arrow={false}>
+      <div>
+        <CenterContent onClick={onClick} />
+      </div>
+    </Popover>
+  )
+}
+
+export function WithTooltip({ onClick }: { onClick: () => void }) {
+  return (
+    <Tooltip title="mParticle Overview" placement="right">
+      <div></div>
+      <CenterContent onClick={onClick} />
+    </Tooltip>
+  )
+}
+
+export function renderHomeButton(props: IGlobalNavigationProps) {
```

Comment:
as you moved this function outside the render function, you could keep it as a component because it will not cause a perf issue anymore.
```suggestion
export function HomeButton(props: IGlobalNavigationProps) {
```